packages/account-v2/src/modules/AddressFields/AddressFields.tsx

```diff
@@ -27,46 +28,22 @@ export const AddressFields = () => {
     } = addressDetailValidations(settings.country_code ?? '', isSvg);
 
     return (
-        <div className='space-y-600 pt-400'>
-            <FormInputField
-                label='First line of address*'
-                name='addressLine1'
-                placeholder='First line of address'
-                validationSchema={addressLine1Schema}
-            />
-            <FormInputField
-                label='Second line of address'
-                name='addressLine2'
-                placeholder='Second line of address'
-                validationSchema={addressLine2Schema}
-            />
-            <FormInputField
-                label='Town/City*'
-                name='addressCity'
-                placeholder='Town/City'
-                validationSchema={addressCitySchema}
-            />
+        <div className='pt-8 space-y-12'>
+            <FormInputField label='First line of address*' name='addressLine1' validationSchema={addressLine1Schema} />
+            <FormInputField label='Second line of address' name='addressLine2' validationSchema={addressLine2Schema} />
+            <FormInputField label='Town/City*' name='addressCity' validationSchema={addressCitySchema} />
             {statesListFetched && statesList.length ? (
                 <FormDropDownField
+                    dropdownIcon={<LabelPairedChevronDownMdRegularIcon />}
```

Comment:
Since this is the only Icon we use for Dropdown, I think we can set it in the `FormDropDownField` component itself?

Reply:
Can can